14.json

```
@@ -1,7 +1,7 @@
 {
-  "$id": "https://schema.spaceapi.io/14-draft.json",
+  "$id": "https://schema.spaceapi.io/14.json",
   "$schema": "http://json-schema.org/draft-07/schema#",
-  "description": "SpaceAPI 0.14",
+  "description": "SpaceAPI 14",
```

I think this would look a bit nicer :slightly_smiling_face: 
```suggestion
  "description": "SpaceAPI v14",
```